src/winconpty/ft_pty/ConPtyTests.cpp

```diff
@@ -2,12 +2,94 @@
 // Licensed under the MIT license.
 
 #include "precomp.h"
+
+// We statically link with winconpty.cpp and so we need to prevent the functions
+// from being marked as __declspec(dllimport) by overriding CONPTY_IMPEXP.
+#define CONPTY_IMPEXP

```

Comment:
for other projects we did this in the project file. I'm OK with doing it here, however, if you want.